### ISSUE TITLE
Cannot boost the mask mAP when training on Maskrcnn-Tensorflow version?

### ISSUE THREAD

**Issue by micZhang123** [GothicAi/Instaboost#16]
Hi, I have tried to use Instaboost according to the demo of YOLACT in Maskrcnn of Tensorflow version. The original model mask mAP is 0.360. All I did is as follows:

```
image = dataset.load_image(image_id)
    aug_flag = np.random.choice([0,1],p=[0.5,0.5])
    if is_train == True and aug_flag == 1:
        target = dataset.image_info[image_id]["annotations"]
        target, image = get_new_data(target, image, None, background=None)
        crowd  = [x for x in target if     ('iscrowd' in x and x['iscrowd'])]
        target = [x for x in target if not ('iscrowd' in x and x['iscrowd'])]
        target += crowd
        dataset.image_info[image_id]["annotations"] = target# need to change
```

But the final mask mAP is not larger than the original one. The largest training result boost I got is 0.5mAP. But I doubt this is owing to the bigger mini-batch. Can you tell me where it goes wrong?
My training params are as follows:

- training dataset: coco-2014
- Instaboost config : default
- Maskrcnn Version: https://github.com/matterport/Mask_RCNN
- Training strategy: Three stage: 4heads, 4heads+final two layers of backbone, All Layers

Any suggestions will be appreciated!
Thanks

**Comment by Fang-Haoshu** [comment]
Hi, how many epochs did you train the network? I guess It should be trained longer.
